### DISCUSSION TITLE
Best way to restart/shutdown Immich?

### DISCUSSION THREAD

**Discussion by DJF3** [immich-app/immich#8231] [Q&A] [answered]
Team,
What is the best way to shutdown or restart Immich? 
`docker stop $(docker ps -a -q)`
Stopping all containers followed by a reboot or shutdown?
Anything that could be initiated from the GUI?

Cheers!
DJ

PS. the Immich app becomes "more awesome" by the day!

**Comment by mmomjian** [comment, marked as answer]
`docker stop` is fine. You can also just restart the host, Docker will handle shutting down and restarting the containers.

**Reply by akostadinov** [reply]
What will happen with the queues and uploaded files that are not yet moved from uploads to library?

What actions are recommended after an unplanned restart or power failure?

P.S. What is Redis used for? I don't see persistent storage configured for it so it will lose all data it holds when container is recreated it seems.

**Reply by GameKiller168** [reply]
<img width="1633" height="284" alt="image" src="https://github.com/user-attachments/assets/574c1e50-ec1c-402b-a9b7-ef116893f693" />
this happens every time i restart my host...

**Reply by Underdoge** [reply]
Having the same issue as @GameKiller168, going to create a separate issue for this.
